suave/gen/suave_spec.yaml

```diff
@@ -475,3 +475,11 @@ functions:
         - name: message
           type: bytes
           description: "Decrypted message"
+  - name: getInsecureTime
+    address: "0x000000000000000000000000000000007770000c"
+    description: "Returns the current Kettle Unix time in seconds."
```

Comment:
would be good to add ", insecure because it assumes trust in Kettle's clock" or something like that to the description plz